engines/saga/saga.h

```diff
@@ -631,8 +678,12 @@ class SagaEngine : public Engine {
 	const char *getTextString(int textStringId);
 	void getExcuseInfo(int verb, const char *&textString, int &soundResourceId);
 
+	void enableKeyMap(int mode);
+
 private:
 
+	int _currentPanelMode;
```

Comment:
You can use `_interface->getMode()`

Reply:
Right. Thanks for the heads up! Yeah, this is a simple way of performing this check